frontend/src/pages/about/PageOutreach.vue

```diff
@@ -0,0 +1,146 @@
+              YouTube channel with lesson recordings.
+            </li>
+            <li>
+              Youtube channel:
+              <a href="https://www.youtube.com/@obo-academy" target="_blank"
+                >https://www.youtube.com/@obo-academy</a
+              >
+            </li>
+            <li>
+              Join the OBO Academy mailing list using
+              <a href="https://groups.google.com/g/obo-academy" target="_blank"
+                >https://groups.google.com/g/obo-academy</a
+              >
+            </li>
+          </ul>
+        </li>
+        <li>
+          <a href="https://mondo.monarchinitiative.org" target="_blank"
+            >The Mondo Disease Ontology (Mondo)</a
+          >
+          <p>
+            Mondo aims to harmonize disease definitions across the world. The
+            name Mondo comes from the latin word ‘mundus’ and means ‘for the
+            world.’
+          </p>
+          <ul>
+            <li>
+              The Mondo Outreach calls aim to bring together stakeholders and
+              clinical terminology experts to discuss specific use cases and
+              requirements. Our goal is to improve Mondo with a focus on our
+              user needs.
+            </li>
+            <li>
+              These calls are scheduled for the second Thursday of every month.
+              You can find more information about the calls
+              <a
+                href="https://mondo.monarchinitiative.org/pages/workshop/#outreach"
+                target="_blank"
+                >here</a
+              >.
+            </li>
+            <li>
+              You can find the schedule with Zoom information
+              <a
+                href="https://docs.google.com/document/d/1MStpDyh9LOZYJTjLtnpOsNYc3HaeU-bz0CHAI9xOjfQ/edit#heading=h.6sqkx1xhumse"
+                target="_blank"
+                >here</a
+              >.
+            </li>
+          </ul>
+        </li>
+        <li>
+          <a href="https://linkml.io/linkml/" target="_blank"
+            >LinkML (the Linked data Modeling Language)</a
+          >
+          <p>
+            LinkML is a flexible modeling language that allows you to author
+            schemas in YAML that describe the structure of your data.
+          </p>
+          <p>
+            LinkML is used to model the Monarch data model, and we welcome your
+            contributions to the LinkML community. You can learn more
+            <a
+              href="https://linkml.io/linkml/faq/contributing.html"
+              target="_blank"
+              >here</a
+            >.
+          </p>
+        </li>
+      </ul>
+    </AppSection>
+    <AppSection>
+      <AppHeading>Follow us to learn more!</AppHeading>
+      <p>
+        Stay updated on the latest from the Monarch Initiative by joining our
+        <a
+          href="https://groups.google.com/a/tislab.org/g/monarch-users"
+          target="_blank"
```

Comment:
Please remove this sentence and link. We are discussing the best way to go about this mailing list and it is not ready for prime time.